hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -595,66 +607,36 @@ class Statistics(IntEnum):
     always = 2
 
 
-class Verbosity(object):
+@unique
+class Verbosity(IntEnum):
+    quiet = 0
+    normal = 1
+    verbose = 2
+    debug = 3
+
+    @staticmethod
+    def _get_default():
+        var = os.getenv('HYPOTHESIS_VERBOSITY_LEVEL')
+        if var is not None:  # pragma: no cover
+            note_deprecation(
+                'The $HYPOTHESIS_VERBOSITY_LEVEL environment variable is '
+                'deprecated, and will be ignored by a future version of '
+                'Hypothesis.  Configure your verbosity level via a '
+                'settings profile instead.'
+            )
+            if var not in [v.name for v in list(Verbosity)]:
```

Comment:
`var not in Verbosity.__members__` perhaps? Or look up `verbosity.__members__[var]` and catch the `KeyError`.